projects/go-lib/src/lib/components/go-toaster/go-toaster.service.ts

```diff
@@ -75,6 +75,7 @@ export class GoToasterService {
     comp.message = toastInterface.message;
     comp.type = toastInterface.type;
     comp.header = toastInterface.header;
+    comp.dismissable = toastInterface.dismissable !== undefined ? toastInterface.dismissable : false;
```

Comment:
We can actually simplify this in JavaScript land:
```ts
comp.dismissable = toastInterface.dismissable;
```
If `dismissable` is `undefined`, `null`, or `false` it will evaluate to `false` and because it's strictly typed to be a `boolean` value it should only ever be set to `true` or default to `false` (whether it's actually `false` or just `null/undefined`).

Reply:
Oh that's awesome! Thanks